include/smack/SmackRep.h

```diff
@@ -78,16 +79,17 @@ class SmackRep {
     : aliasAnalysis(aa), naming(N), program(P),
       targetData(aa->getDataLayout()), globalsBottom(0) {
     uniqueFpNum = 0;
+    ptrSizeInBits = targetData->getPointerSizeInBits();
   }
   DSAAliasAnalysis* getAliasAnalysis() { return aliasAnalysis; }
   Program& getProgram() { return program; }
 
 private:
-  void addInit(unsigned region, const Expr* addr, const llvm::Constant* val);
+  void addInit(unsigned region, const Expr* addr, const llvm::Constant* val, const llvm::GlobalValue* V, bool safety);
 
-  const Expr* pa(const Expr* base, int index, int size);
-  const Expr* pa(const Expr* base, const Expr* index, int size);
-  const Expr* pa(const Expr* base, const Expr* index, const Expr* size);
+  const Expr* pa(const Expr* base, int index, int size, int i_size = 0, int t_size = 0);
+  const Expr* pa(const Expr* base, const Expr* index, int size, int i_size = 0, int t_size = 0);
+  const Expr* pa(const Expr* base, const Expr* index, const Expr* size, int i_size = 0, int t_size = 0);
   
   const Expr* b2p(const llvm::Value* v);
```

Comment:
Actually they are not obsolete because even though `$pa` is renamed to `$add.ref`. These functions which converts `getElementPtr` to `$add.ref` keeps old names.